src/constructs/ec2/security-groups/base.test.ts

```diff
@@ -12,6 +12,17 @@ describe("The GuSecurityGroup class", () => {
     publicSubnetIds: [""],
   });
 
+  it("applies the Stage, Stage and App tags", () => {
```

Comment:
Minor correction: 

```suggestion
  it("applies the Stack, Stage and App tags", () => {
```

Reply:
Argh! I thought I crafted this changeset so well!